lib/ordered_map_component.dart

```diff
@@ -61,6 +63,12 @@ class OrderedMapComponent extends IsometricTileMapComponent {
     return false;
   }
 
+  void removeUnit(Unit unit) {
+    occupiedBlocks.remove(unit.block);
```

Comment:
is occupiedBlocks updated when units move?

Reply:
They are only occupied when they stop at a block, they are not updated while walking. So it is used for like reserving a spot when they start walking so that the algorithm for moving many units to an area gets simpler.